src/compounds/BlankRequestForm/BlankRequestForm.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import {
+  Button,
+  Form,
+  Row,
+  Col,
+} from 'react-bootstrap'
+import Title from '../../components/Title/Title'
+import ShippingDetails from './ShippingDetails'
+import AdditionalInfo from './AdditionalInfo'
+
+const BlankRequestForm = () => {
+  const initialState = {
+    name: 'New Request',
+    billingSameAsShipping: false,
+    proposedDeadline: null,
+    billing: {
+      id: null,
+      street: '',
+      street2: '',
+      city: '',
+      state: '',
+      zipCode: '',
+      country: '',
+      text: '',
+    },
+    shipping: {
+      street: '',
+      street2: '',
+      city: '',
+      state: '',
+      zipCode: '',
+      country: '',
+      text: '',
+    },
+    data: {
+      timeline: '',
+      description: '',
+      suppliersIdentified: 'Yes',
+    },
+    // TODO(alishaevn): how do we handle attachments?
+  }
+
+  const [requestForm, setRequestForm] = useState(initialState)
+
+  /**
+   * @param {object} event onChange event
+   * @param {string} property dot notated string representing the property in initialValue
+   * @returns {object} the updated component state
+   */
+  const updateRequestForm = (value, property) => {
+    const [initialProperty, nestedProperty] = property.split('.')
+
+    setRequestForm((currentState) => {
+      const updatedState = nestedProperty
+        ? { [initialProperty]: { ...requestForm[initialProperty], [nestedProperty]: value } }
+        : { [initialProperty]: value }
+
+      return {
+        ...currentState,
+        ...updatedState,
+      }
+    })
+  }
+
+  const handleSubmit = (event) => {
+    event.preventDefault()
+    if (requestForm.billingSameAsShipping === true) {
+      Object.assign(requestForm.billing, requestForm.shipping)
```

Comment:
```suggestion
     updateRequestForm(requestForm.shipping, 'requestForm.billing')
```

Reply:
the updating of billing takes place after the form is submitted. will return to this thought another time.